README.md

```diff
@@ -6,22 +6,40 @@ Puppet module for managing Windows poweshell DSC resources.
 
 This module generates Puppet Types based on DSC resources MOF schema files.
 
-In this version, the following DSC Resources are already build and ready for usage:
-- All base DSC resources found in powershell 4. ([wmf 4.0](http://www.microsoft.com/en-us/download/details.aspx?id=40855))
-- All DSC resources found in the [DSC resource kit wave 6](http://gallery.technet.microsoft.com/DSC-Resource-Kit-All-c449312d).
+In this version, the following DSC Resources are already built and ready for use:
+- All base DSC resources found in PowerShell 5. ([WMF 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=46889))
+- All DSC resources found in the [DSC Resource Kit Wave 10](https://gallery.technet.microsoft.com/scriptcenter/DSC-Resource-Kit-All-c449312d).
 
-This module is available on the [Puppet Forge](https://forge.puppetlabs.com/msutter/dsc)
+This module is available on the [Puppet Forge](https://forge.puppetlabs.com/puppetlabs/puppetlabs-dsc)
 
 ## Windows systems prerequisites
- - Powershell 4 which is included in the [Windows Management Framework 4.0](http://www.microsoft.com/en-us/download/details.aspx?id=40855)
- - [DSC resource kit wave 6](http://gallery.technet.microsoft.com/DSC-Resource-Kit-All-c449312d) must be installed to use it's own resources.
+ - PowerShell 5 which is included in the [Windows Management Framework 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=46889). PowerShell v5 is currently in limited preview, so the above link may change after official release.
 
 ## Installation on your puppet master
-    puppet module install msutter-dsc
+    puppet module install puppetlabs-dsc
 
 ## Usage
-You can use DSC resource by prefixing resource names and parameters with 'dsc_'.
-The resulting resource name has to be in lowercase. e.g: 'dsc_windowsfeature'.
+
+Before running a Puppet manifest with DSC Resources, you must set the Local Configuration Manager (LCM) `RefreshMode` to `Disabled` so that Puppet can manage the target node with DSC. This is a PowerShell DSC requirement and only has to be done once per system.
+
+This can be done with our built-in Puppet type:
+
+```
+dsc::lcm_config {'disable_refreshmode':
+  refresh_mode => 'Disabled',
+}
+```
+
+You can then use a DSC Resource by prefixing each DSC Resource name and parameter with 'dsc_'.
+
+```
+dsc_windowsfeature {'IIS':
+  dsc_ensure => 'present',
+  dsc_name   => 'Web-Server',
+}
+```
+
+All DSC Resource names and parameters have to be in lowercase. e.g: `dsc_windowsfeature` or `dsc_ensure`.
```

Comment:
Lets leave out additional mention of `dsc_ensure` on this line given we're not sure if we want / need `dsc_ensure` in addition to the standard Puppet `ensure`